Intersections_3/include/CGAL/Intersections_3/internal/Bbox_3_Triangle_3_do_intersect.h

```diff
@@ -411,7 +413,8 @@ bool do_intersect_bbox_or_iso_cuboid(const typename K::Triangle_3& a_triangle,
                                        const FT& c_alpha,
                                        const FT& c_beta) -> Uncertain<Sign>
   {
-    return CGAL::sign(- c_alpha * alpha + c_beta * beta);
+    Sgn<FT> sign;
+    return sign(- c_alpha * alpha + c_beta * beta);
```

Comment:
For my curiosity, why does that change anything?

`CGAL::sign` is implemented that way:
https://github.com/CGAL/cgal/blob/8c9fc331cd0e523f897f830da480483b6fb70d5d/Algebraic_foundations/include/CGAL/number_utils.h#L240-L247
and `CGAL::Sgn` is implemented that way:
https://github.com/CGAL/cgal/blob/8c9fc331cd0e523f897f830da480483b6fb70d5d/Algebraic_foundations/include/CGAL/number_utils_classes.h#L58-L59

so in the end they both use `Real_embeddable_traits<NT>::Sgn`...

Reply:
Side note: we could reimplement `CGAL::Sgn` (and all the others) as template aliases, to avoid an extra template instantiation.

Reply:
If I remember correctly for the testsuite with Leda numbers when not qualified it uses an overload coming from Leda which has a different return type.

Reply:
Might as well use `compare` instead of `sign(x-y)`.

Reply:
Bad grep

Reply:
> If I remember correctly for the testsuite with Leda numbers when not qualified it uses an overload coming from Leda which has a different return type.

Here the commit https://github.com/CGAL/cgal/pull/5767/commits/b67589e16b9db85d0a66bbeb6e2c4024e67a03bf replaced `CGAL::sign` with the use of `CGAL::Sgn`.